lib/tessel/deploy.js

```diff
@@ -225,8 +225,11 @@ actions.runScript = function(t, filepath, entryPoint) {
           resolve();
         }
 
-        // Once we get a SIGINT from the console, stop the script
-        process.once('SIGINT', scriptStopped);
+        // Once we get a SIGINT from the console
+        process.once('SIGINT', function() {
+          // pass the SIGINT to the remote process
+          remoteProcess.stdout.signal('SIGINT');
+        });
```

Comment:
Also also, I'll need to make the SIGINT handler non-anonymous so that [this line](https://github.com/tessel/t2-cli/pull/260/files#diff-191f4ebe176e9e816abc0445a664be98R237) can effectively remove the event listener.